pkg/ipamd/ipamd.go

```diff
@@ -1807,7 +1796,7 @@ func (c *IPAMContext) isDatastorePoolTooHigh() bool {
 }
 
 func (c *IPAMContext) warmPrefixTargetDefined() bool {
-	return c.warmPrefixTarget >= noWarmPrefixTarget && c.enableIpv4PrefixDelegation
+	return c.warmPrefixTarget >= defaultWarmPrefixTarget && c.enableIpv4PrefixDelegation
```

Comment:
will we ever change `defaultWarmPrefixTarget` to a value than `0`?
If we change it to be same thing like 2, then this read as `c.warmPrefixTarget >= 2`. I feel this should be something like `c.warmPrefixTarget >= 0`

Reply:
`defaultWarmPrefixTarget ` is 1 in this PR and also setting just WARM_PREFIX_TARGET = 0 won't be supported. If someone sets `WARM_IP_TARGET = num` and `WARM_PREFIX_TARGET = 0 or any num` then WARM_IP_TARGET will take precedence.